src/sx/package.json

```diff
@@ -4,7 +4,7 @@
   "homepage": "https://github.com/adeira/universe/tree/master/src/sx",
   "license": "MIT",
   "private": false,
-  "version": "0.15.1",
+  "version": "0.15.2",
```

Comment:
I think the removal of the CSS reset will be released when this is bumped, so better make it `0.16.0`

Reply:
OK, done

Reply:
In general, I'd say releasing patch versions for 0.x is not necessary - feel free to always bump the minor. Semver understands these versions a bit differently and more aggressively:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

Also in FAQ:

> How should I deal with revisions in the 0.y.z initial development phase?
>
> The simplest thing to do is start your initial development release at 0.1.0 and then increment the minor version for each subsequent release.

But up to you, ofc. https://semver.org/